test/e2e/config_test.go

```diff
@@ -29,7 +29,7 @@ rebaseRules:
   resourceMatchers:
   - kindNamespaceNameMatcher:
       kind: ConfigMap
-      namespace: kapp-test
+      namespace: ` + env.Namespace + `
```

Comment:
What do you think about using `fmt.Sprintf()` to format it?

Reply:
That's actually a cleaner approach. `strings` would also work but `fmt.Sprintf()` would be more readable

Reply:
I guess we could use `fmt` when we need to add the namespace to one or two places, otherwise `strings.Replace` would be a good option.
Also, I think it would be good to use something like `<e2e-test-ns>` instead of `kapp-test` and then replace it with `env.Namespace` so that it is evident while reading the yaml that this value will be replaced.

Reply:
There are a couple of other places where you might wanna make this change :)